pkg/user/user_integration_test.go

```diff
@@ -322,11 +316,10 @@ func TestUserHandler(t *testing.T) {
 				t.Log("RefreshTokensUsingCookie")
 
 				_, email, password := createUser(t, client, userService)
-				var tokens *token.Tokens
-				client.PostJSON(t, "/tokens", nil, &tokens, inttest.WithBasicAuth(email, password))
-				require.NotEmpty(t, tokens.AccessToken, "should return an access token")
+				accessToken, refreshToken := client.SignIn(t, email, password)
+				require.NotEmpty(t, accessToken.Value, "should return an access token")
```

Comment:
Could we move this require inside of `client.SignIn`? I assume it should never be empty after signin right?